bids-validator/bids_validator/rules/file_level_rules.json

```
@@ -451,7 +453,9 @@
         "_channels\\.tsv",
         "_ieeg\\.json",
         "_coordsystem\\.json",
-        "_photo\\.jpg"
+        "_photo\\.jpg",
+        "_photo\\.png",
+        "_photo\\.tif"
```

```suggestion
        "_photo\\.jpg"
```
Same as above, this change does not apply to "ieeg".

---

Done